File: compMine/src/main/debug/java/com/effective/android/component/mine/TestActivity.java
package com.effective.android.component.mine;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.effective.android.base.util.FragmentUtils;
import com.plugin.component.ComponentManager;
import com.plugin.component.SdkManager;


public class TestActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.mine_activity_main);
        ComponentManager.init(getApplication());
        ComponentMineSdk componentMineSdk = SdkManager.getSdk(ComponentMineSdk.class);
        FragmentUtils.add(getSupportFragmentManager(), R.id.id_fragment_container, componentMineSdk.getMainFragment(), "");
    }
}

File: compMine/src/main/debug/java/com/effective/android/component/mine/TestActivity.kt
package com.effective.android.component.mine

import android.os.Bundle

import com.effective.android.base.util.FragmentUtils
import com.effective.android.debug.DebugActivity
import com.plugin.component.ComponentManager
import com.plugin.component.SdkManager


class TestActivity : DebugActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        ComponentManager.init(application)
        val componentMineSdk = SdkManager.getSdk(ComponentMineSdk::class.java)
        FragmentUtils.add(supportFragmentManager, R.id.id_fragment_container, componentMineSdk!!.getMainFragment(), "")
    }
}
